### ISSUE TITLE
tflite_runtime pip package built using bazel is missing `metrics_portable` module

### ISSUE THREAD

**Issue by avroshk** [tensorflow/tensorflow#49198]
<em>Please make sure that this is a build/installation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:build_template</em>

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu 18.04.5 LTS (running in docker built from `latest-devel`)
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: N/A
- TensorFlow installed from (source or binary): Source `master` branch
- TensorFlow version: 2.6.0 
- Python version: 3.6
- Installed using virtualenv? pip? conda?: pip
- Bazel version (if compiling from source): 3.7.2
- GCC/Compiler version (if compiling from source): gcc version 7.5.0
- CUDA/cuDNN version: N/A
- GPU model and memory: N/A

## Description 
* Standalone tflite_runtime pip package built using bazel ([reference](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/tools/pip_package#normal-build-for-your-workstation)) is missing `metrics_portable` module that is imported in `tflite_runtimer/interpreter.py`
* The missing module is only encountered when the package is built using bazel. `build_pip_package.sh` bash script and cmake script include the metrics_* modules that are missing from the bazel scripts (refer: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/tools/pip_package/build_pip_package.sh#L35-L36)

## Steps to reproduce
* Steps to reproduce the issue are detailed in this repo: https://github.com/avroshk/build-tflite-runtime

## Error logs
```
tflite_runtime==2.6.0
Traceback (most recent call last):
  File "test_interpreter.py", line 4, in <module>
    import tflite_runtime.interpreter as tflite
  File "/usr/local/lib/python3.6/dist-packages/tflite_runtime/interpreter.py", line 42, in <module>
    from tflite_runtime import metrics_portable as metrics
ImportError: cannot import name 'metrics_portable'
```

**Comment by bhack** [comment]
Do you want to submit a PR?

**Comment by abattery** [comment]
@yyoon could you take a look?

**Comment by PINTO0309** [comment]
It's unofficial, but I applied the patch and built it, and it seemed to work fine.
https://github.com/PINTO0309/TensorflowLite-bin

**Comment by avroshk** [comment]
Addressed in PR #49199
